### DISCUSSION TITLE
没有票时间选项

### DISCUSSION THREAD

**Discussion by hzw123as** [mikumifa/cppTickerBuy#7] [提问专区]
![屏幕截图 2024-09-16 191047](https://github.com/user-attachments/assets/830305b7-07d8-49df-a017-48fe13927743)

**Comment by hzw123as** [comment]
大佬这是我的电脑问题吗

**Reply by hzw123as** [reply]
是浏览器的问题，微软浏览器不行

**Reply by SkyFall5Yzy** [reply]
为啥我谷歌也不行啊？
